fast-markdown-mcp/src/fast_markdown_mcp/server.py

```
@@ -221,6 +242,205 @@ async def search_files(self, query: str) -> str:
+**Match Type:** {match_type_display}
+**Confidence:** {confidence_percent}%
+
+**Snippet:**
+{match["snippet"]}
+
+**Citation:** `{match["file_id"]}#{match["section_id"]}`
+
+---""")
+            
+            return "\n".join(results)
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)